puzzles/models.py

```diff
@@ -40,6 +41,10 @@ class Puzzle(models.Model):
     )
     url = models.URLField(blank=True)
 
+    created_on = models.DateTimeField(auto_now_add=True)
+    updated_on = models.DateTimeField(auto_now=True)
+    solved_on = models.DateTimeField(default=None, null=True, blank=True)
```

Comment:
How is "solved_on" going to be set? In my mind, it can either be:
1) the "created_on" timestamp of the last Answer object for this puzzle that is marked as CORRECT
2) the "updated_on" timestamp of the last Answer object for this puzzle that is marked as CORRECT

On a related note, if we're just using one of the Answer timestamps, I wonder if having this as a model object is really worth it (we'd be putting the same information in 2 different places in the DB). It might just be better to have a "solved_time()" method that looks at the answers for the puzzle.

Reply:
Option 1 is probably the way to go; I can't imagine that option 2 would be different unless we make a typo, in which case option 1 contains the time we want.  (Unless we enter in a "partial", but that wouldn't be marked as correct, and the eventual correct answer would be a different guess anyway.)

That sounds really sensible, so I'll do that instead.